### PR TITLE
Fix/121 PHP Version Check

### LINKED ISSUE #121: Add PHP checks
More info: https://github.com/10up/convert-to-blocks/pull/123#issuecomment-1611475826

### PR DESCRIPTION
**Pull request by bmarshall511**
### Description of the Change

Added a minimum PHP version check for 7.4.

Closes #121

### How to test the Change

1. Spin up the site with a version of PHP below 7.4
2. Verify an admin notice has been added alerting the user of the failed check
3. Verify the admin nav doesn't include any of the custom post type options or settings

### Changelog Entry

> Added - >=PHP 7.4 version check

### Credits

Props @bmarshall511 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: eight-day-week.php [hunk 1]
```diff
@@ -60,6 +60,49 @@
 define( 'EDW_ARTICLE_STATUS_TAX', 'pi-article-status' );
 define( 'EDW_AJAX_NONCE_SLUG', 'edw_ajax_nonce' );
 
+/**
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @return string Minimum version required.
+ */
+function minimum_php_requirement() {
```
Comment (iamdharmesh) [inline eight-day-week.php]: Maybe add the prefix `edw_` here as we don't have a namespace in this file? Or add a namespace?

**Review by iamdharmesh — CHANGES_REQUESTED** [review]
Thanks for the PR @bmarshall511. Added a minor note to discuss.

Thank you.

File: eight-day-week.php (hunk 1 above)
Comment (bmarshall511) [reply eight-day-week.php]: @iamdharmesh Nice catch, updated accordingly!

**Review by iamdharmesh — DISMISSED** [review]
Thanks for the changes @bmarshall511. LGTM! 

@dkotter it seems the PR branch checked out from the `trunk` branch, so it shows some commits of the `trunk` branch in diff. Do you see any disadvantages of merging this to develop a branch?

**Review by dkotter — APPROVED** [review]

**Pull request merged**